<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="浪里个浪" ignored="2" name="examplel" tests="6" failures="1" timestamp="03 八月 2022 01:35:51 GMT" time="3.355" errors="0">
  <testcase name="wo404" time="0.853" classname="com.hxk.online.CourseRecord404">
    <failure type="java.lang.AssertionError" message="服务器返回的状态码不是200 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: 服务器返回的状态码不是200 expected [200] but found [404]
at com.hxk.online.CourseRecord404.wo404(CourseRecord404.java:61)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- wo404 -->
  <testcase name="index" time="0.238" classname="com.hxk.online.course.VideoList"/>
  <testcase name="nchannel_list" time="0.417" classname="com.hxk.online.sale.Nchannel_list"/>
  <testcase name="doneUpload" time="0.783" classname="com.hxk.online.course.DoneUpload"/>
  <testcase name="index" time="0.211" classname="com.hxk.online.Index"/>
  <testcase name="videoDel" time="0.632" classname="com.hxk.online.course.VideoDel"/>
  <testcase name="test2" time="0.0" classname="com.hxk.online.TestDemo2">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="com.hxk.online.TestDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- examplel -->
